ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-osuosl.yaml

```diff
@@ -0,0 +1,758 @@
+kind: Template
```

Comment:
This adds a new template which is 100% undesirable, as we are now actively working on deprecating that mechanism and migrating jobs that use it to [multi-stage](https://steps.ci.openshift.org/help) tests.

Is something actively blocking you from using multi-stage tests to implement this functionality? We definitely do not want to merge a new template and jobs using it, unless there is a known blocker.